src/Microsoft.AspNet.OData.Shared/Formatter/ODataResourceSetItemBase.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.OData;
+
+namespace Microsoft.AspNet.OData.Formatter.Deserialization
```

Comment:
>Microsoft.AspNet.OData.Formatter.Deserialization [](start = 10, length = 48)

This file is currently in src/Microsoft.Aspnet.OData.Shared/Formatter. Seems like it should be in the /Deserialization subfolder of that folder, along with the other related classes?